packages/docs/src/page-configs/getting-started/kitchensink/page-config.ts

```diff
@@ -0,0 +1,36 @@
+import { ApiDocsBlock, BlockType } from '@/types/configTypes'
+import { shallowReactive } from 'vue'
+
+const importedConfigsContext = require.context(
+  '@/page-configs/ui-elements/',
+  true,
+  /page-config.ts$/,
+  'lazy',
+)
+
+export const config = shallowReactive<ApiDocsBlock[]>([])
+
+importedConfigsContext.keys().forEach(async (filename) => {
+  let importedConfig = null
+  try {
+    importedConfig = (await importedConfigsContext(filename)).default
+  } catch {
+    return
+  }
+
+  config.push({ ...importedConfig[0], type: BlockType.SUBTITLE }) // title
+
+  const example =
+    importedConfig.find(
+      (blockElement: any) =>
+        blockElement.type === BlockType.EXAMPLE &&
+        blockElement.component === 'Default',
+    ) ||
+    importedConfig.find(
```

Comment:
What's the point of this rollback?

Reply:
We want to grab _only_ default examples? And if there is no default example don't show component at all? So then we need to create default examples for all our components (right now some components don't have it)

Reply:
> right now some components don't have it

Let's figure out which components don't have it and think about it. :)

Why would we have components without default example?

Reply:
Only default examples now, no exmaple === don't show